code/test/events.js

```diff
@@ -0,0 +1,172 @@
+// Test all events emitted
```

Comment:
Cleanup? Also this does not appear to be testing event emissions. Perhaps remove and add in later PRs after it is implemented?